### DISCUSSION TITLE
i2c oled

### DISCUSSION THREAD

**Discussion by 14lfl** [kachurovskiy/nanoels#188] [Q&A] [answered]
Hello everyone, can we use an i2c oled display on the H2 version by installing the appropriate library.
THANKS

**Comment by kachurovskiy** [comment]
Hi @14lfl, we used to have I2C support in H1 but it wasn't carried over into H2. You can see how it's done in https://github.com/kachurovskiy/nanoels/blob/ee9b80e4edecbee13295d2b295a7f7feb30526c3/h1/NanoEls.ino#L32 and make the necessary edits in your local h2.ino version. Cheers!

**Reply by 14lfl** [reply]
Thank you for this quick feedback.  I was afraid of using an Oled I2C and lowering the performance of the nano.  I don't have a 200 step driver.....

**Reply by kachurovskiy** [reply, marked as answer]
It will not affect the speed of the controller since display is only updated when the stepper isn't running.

**Reply by 14lfl** [reply]
Thanks for all the details.  Have a good evening
